src/assembly/development.xml

```diff
@@ -12,6 +12,16 @@
     <dependencySets>
         <dependencySet>
             <outputDirectory>share/java/kafka-connect-hdfs/</outputDirectory>
+            <useProjectArtifact>true</useProjectArtifact>
+            <useTransitiveFiltering>true</useTransitiveFiltering>
+            <excludes>
```

Comment:
Let's just add a quick comment here to explain why we're making these exclusions. Just needs enough that someone like Michael who comes to look at this for packaging will understand why this is here. Same goes for `package.xml`.